x11/rxvt-unicode/Portfile

```
@@ -24,38 +24,39 @@ master_sites    http://dist.schmorp.de/rxvt-unicode/ \
                 http://dist.schmorp.de/rxvt-unicode/Attic/
 use_bzip2       yes
 
-checksums       rmd160  ce20bcda75cadd4a22edf166ce50384080de1b23 \
-                sha256  643116b9a25d29ad29f4890131796d42e6d2d21312282a613ef66c80c5b8c98b \
-                size    904950
+checksums       rmd160  a2201a7ff63283715fe7230e17291b5c12386c1b \
+                sha256  fe1c93d12f385876457a989fc3ae05c0915d2692efc59289d0f70fabe5b44d2d \
+                size    877186
 depends_build \
         port:pkgconfig
 
 # urxvt also depends on xorg-libX11, xrender and fontconfig but all these ports are in the dependency tree of Xft2
 depends_lib \
+        port:libptytty \
         port:Xft2 \
         port:startup-notification \
         port:xorg-libXmu \
         port:ncurses \
-        port:perl5.30
+        port:perl5.34
 
 # clang++ 3.0 produces unstable urxvt binary which sometimes causes errors like:
 #        "terminate called throwing an exceptionAbort trap: 6"
 compiler.blacklist {clang < 318}
 
-configure.perl  ${prefix}/bin/perl5.30
+configure.perl  ${prefix}/bin/perl5.34
 
 patchfiles      patch-Makefile.in.diff \
-                patch-no-clock_gettime.diff \
                 patch-perl-embed-args.diff
 
+configure.cxxflags-append -std=c++11
```

If a compiler supporting C++11 is required then `compiler.cxx_standard 2011` should also be set:

```suggestion
compiler.cxx_standard 2011
configure.cxxflags-append -std=c++11
```

---

Added, thanks.